build_docs.sh

```diff
@@ -28,29 +28,27 @@ doc_files=(
 	'Blackbox.md'
 	'Migrating from baseflight.md'
 	'Boards.md'
-	'Board - AlienFlight.md'
-	'Board - CC3D.md'
-	'Board - CJMCU.md'
-	'Board - Naze32.md'
-	'Board - Sparky.md'
-	'Board - Olimexino.md'
-	'Board - ChebuzzF3.md'
+	'./boards/Board - AlienFlight.md'
+	'./boards/Board - CC3D.md'
+	'./boards/Board - CJMCU.md'
+	'./boards/Board - Naze32.md'
+	'./boards/Board - Sparky.md'
+	'./boards/Board - Olimexino.md'
+	'./boards/Board - ChebuzzF3.md'
 )
 
 if which gimli >/dev/null; then
-	echo "Building ${filename}.pdf"
+	echo "Building ${filename}"
 	pushd . >/dev/null
 	cd docs
-
-	rm -f ${filename}.md
+	rm -f $filename
```

Comment:
This is a markdown file. Why are you not using the .md extension to make this explicit?